src/components/Tabs/Tabs.tsx

```diff
@@ -0,0 +1,87 @@
+import * as RadixTabs from "@radix-ui/react-tabs";
+import styled from "styled-components";
+
+export type TabProps = {
+  label: string;
+  value: string;
+  children?: React.ReactNode;
+};
+
+export type TabsProps = {
+  defaultValue?: string;
+  ariaLabel?: string;
+  onValueChange?: (s: string) => unknown;
+  children: React.ReactElement<TabProps>[] | React.ReactElement<TabProps>;
+};
+
+const Trigger = styled(RadixTabs.Trigger)`
+  border: none;
+  padding: ${props =>
+    `${props.theme.click.tabs.space.y} ${props.theme.click.tabs.space.x}`};
+
+  border-top-left-radius: ${props => props.theme.click.tabs.radii.all};
+  border-top-right-radius: ${props => props.theme.click.tabs.radii.all};
+
+  border-bottom: ${props =>
+    `${props.theme.border.width[2]} solid ${props.theme.click.tabs.basic.color.accent.hover}`};
```

Comment:
@gjones is `theme.border.width[2]` the correct token here for the size of the border bottom when the tab is selected?